examples/benchmark.py

```diff
@@ -183,51 +183,74 @@ def benchmark_train_and_valid(base_dir,train_dataset,valid_dataset,tasks,
                                             verbosity=verbosity,
                                             mode="classification")
   
-  assert model in ['graphconv', 'tf', 'rf','logreg']
+  assert model in ['graphconv', 'tf', 'tf_robust', 'rf','logreg']
 
   if model == 'tf':
-    # Initialize model folder
-    model_dir_tf = os.path.join(base_dir, "model_tf")
-    
-      # Building tensorflow MultiTaskDNN model
+    # Building tensorflow MultiTaskDNN model
     dropouts = hyper_parameters['dropouts']
     learning_rate = hyper_parameters['learning_rate']
     layer_sizes = hyper_parameters['layer_sizes']
     batch_size = hyper_parameters['batch_size']
     nb_epoch = hyper_parameters['nb_epoch']
 
-    tensorflow_model = dc.models.TensorflowMultiTaskClassifier(len(tasks),
-          n_features,  learning_rate=learning_rate, layer_sizes=layer_sizes, 
+    model_tf = dc.models.TensorflowMultiTaskClassifier(len(tasks),
```

Comment:
One question here is that if we just use the default weight_init_stddevs and bias_init_consts, it will be hard to define a more than one hidden layer structure. Should we add that it or just using one layer is good enough here? Actually I have already done that on my local branch, I can initiate another PR about it.